src/syscheckd/db/tests/fimDBHelper/fimDBHelperTest.cpp

```diff
@@ -43,7 +43,7 @@ TEST_F(FIMHelperTest, insertItemToDatabase)
 TEST_F(FIMHelperTest, deleteItemToDatabase)
 {
     std::string tableName;
-    nlohmann::json filter;
+    std::string filter;
```

Comment:
This test is failling because filter and tableName will be null. You need to initialize this variables.

Reply:
This change is already merged into 9103.